src/System.Private.CoreLib/src/System/EETypePtr.cs

```diff
@@ -235,5 +235,15 @@ internal RuntimeImports.RhCorElementTypeInfo CorElementTypeInfo
                 return RuntimeImports.GetRhCorElementTypeInfo(corElementType);
             }
         }
+
+        [Intrinsic]
```

Comment:
Could you please put it under `#if CORERT` ? nutc does not like things that it does not know about to be marked as intrinsics ... yet.